thingsboard_gateway/connectors/opcua/opcua_connector.py

```diff
@@ -222,10 +222,58 @@ def on_attributes_update(self, content):
     def server_side_rpc_handler(self, content):
         try:
             rpc_method = content["data"].get("method")
+
+            # firstly check if a method is not service
+            if rpc_method == 'set' or rpc_method == 'get':
+                args_list = content['data']['params'].split(';')
+                try:
+                    ns, i = [item.split('=')[-1] for item in args_list[0:2]]
+                except IndexError:
+                    log.error('Not enough arguments. Expected min 2.')
+                    self.__gateway.send_rpc_reply(content['device'],
+                                                  content['data']['id'],
+                                                  {content['data']['method']: 'Not enough arguments. Expected min 2.',
+                                                   'code': 400})
+
+                node_list = []
+                self.__search_node(current_node=content['device'], fullpath=f'ns={ns};i={i}', result=node_list)
+
```

Comment:
Node identifier can be not only i, it may be also like a path or with different type:

```
i | NUMERIC (UInteger)
s | STRING (String)
g | GUID (Guid)
b | OPAQUE (ByteString)
```